scripts/load_black_spots.py

```diff
@@ -0,0 +1,112 @@
+"""Loads mock black spot data"""
+import argparse
+from datetime import datetime
+import logging
+import json
+import requests
+import sys
+
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger()
+
+
+def get_record_type(api_url, headers, label):
+    """Gets the record type"""
+    response = requests.get('{}/recordtypes/?active=True&limit=all'.format(api_url),
+                            headers=headers)
+    response.raise_for_status()
+    results = response.json()['results']
+    rectype_id = [rt['uuid'] for rt in results if rt['label'] == label][0]
+    logger.info('record type id: {}'.format(rectype_id))
+    return rectype_id
+
+
+def read_black_spots(input_path):
+    """Reads the input json file of black spots"""
+    with open(input_path) as json_file:
+        black_spots_json = json.load(json_file)
```

Comment:
It'd be good to add exception handling here for cases where the JSON isn't valid; there was a minor syntax error in the `black_spots.json` file (I fixed it on the fileshare), and the default JSON exception could be friendlier. Something like `Error parsing JSON: <original exception message>` would make it clear what's happening.